gokart/info.py

```diff
@@ -25,16 +25,27 @@ def make_tree_info(task, indent='', last=True, details=False):
         indent += '|  '
     name = task.__class__.__name__
     result += f'({is_complete}) {name}[{task.make_unique_id()}]'
+
+    if abbr:
+        task_set = task_set or {}
```

Comment:
@vaaaaanquish 
[nits]
`task_set` seems not to be descriptive. How about `visited_tasks`, `visited_task_set` or so.